pennylane/qnodes/jacobian.py

```diff
@@ -318,12 +332,12 @@ def _pd_finite_diff(self, idx, args, kwargs, **options):
         Returns:
             array[float]: partial derivative of the node
         """
-        y0 = options.get("y0", None)
         h = options.get("h", self.h)
         order = options.get("order", 1)
```

Comment:
```suggestion
        order = options.get("order", self.order)
```
(although this might be redundant?